src/functions.cpp

```diff
@@ -1,10 +1,9 @@
-#include "../inc/functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
-#include <algorithm>
+#include "functions.hpp"
 #include <iostream>
-#include <limits>
+#include <algorithm>
```

Comment:
Please sort headers.

Reply:
ok, thanks. I did it before watching last video